data/campaigns/The_Rise_Of_Wesnoth/scenarios/09_Fallen_Lich_Point.cfg

```diff
@@ -344,7 +344,7 @@
         [/filter]
         [message]
             speaker=narrator
-            message= _ "This monolith was erected by me, ― (<i>chipped away</i>), first Mage of the good people of the Green Isle. By its power the Lich-Lord is bound in stone. To end the spell a noble of the line of Kings should utter the following..."
+            message= _ "This monolith was erected by me, ― (<i>chipped away</i>), First Mage of the good people of the Green Isle. By its power, the Lich-Lord is bound in stone. To end the spell, a noble of the line of kings should utter the following..."
```

Comment:
I'm translating this, and I want to check my understanding of it, because I think the meaning has subtly changed:
* "First Mage" would be either the highest ranking mage in the army, or the one appointed to the Royal Court.
* "first Mage" would specifically be the first (chronologically) to occupy that role.

So the "first Mage" would also be the "First Mage", but their successor would be "second First Mage".

Reply:
I think the lore and also TRoW is slated to have an upcoming rework? Meaning you can probably choose your interpretation since it is yet to be finalized.

Reply:
I think I suggested this capitalisation. It's difficult to know what the intent of the original text was. My objection was the capitalisation of Mage on its own, to me it didn't make sense to do so. It would only make sense if First Mage was a title of some sort otherwise it should all be lower-case as in 'first mage of the good people...'. Whether the First was meant as the chronological first mage of the Green Isle, or First as in a position of primacy (eg like 'First Lady' in the US), I don't really know.

As max mentioned, it's subject to being rewritten anyway. Personally I read it as the chronological First Mage, as in, whoever inscribed this was the first formally-recognised magic wielder and there isn't necessarily an implication that it's a position of authority that gets handed down to successors.